doc/source/cd/background.ipynb

```diff
@@ -0,0 +1,608 @@
+    "\n",
+    "```python\n",
+    "label_detector = MMDDrift(y_train.reshape(-1,1), backend='tensorflow', p_val=.05)\n",
+    "y_pred = clf.predict(X_test)\n",
+    "label_detector.predict(y_pred.reshape(-1,1))\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "source": [
+    "#### Concept drift \n",
+    "\n",
+    "Concept drift detectors are coming soon!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Learned drift detection <a class=\"anchor\" id=\"learned\"></a>\n",
+    "\n",
+    "\n",
+    "\n",
+    "- [Classifier](methods/classifierdrift.ipynb)\n",
+    "- [Spot the difference](methods/spotthediffdrift.ipynb)\n",
+    "- [Learned Kernel](methods/learnedkerneldrift.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Online drift detection <a class=\"anchor\" id=\"online\"></a>\n",
+    "\n",
+    "So far, we have discussed drift detection in an *offline* context, with the entire dataset $\\{\\mathbf{X}_i\\}_{i=1}^{N}$ compared to the reference dataset $\\{\\mathbf{X}^{ref}_i\\}_{i=1}^{M}$. However, during model deployment data usually arrives sequentially, and drift detection must be performed in an *online* fashion. \n",
+    "\n",
+    "<div style=\"text-align: center\">\n",
+    "<img src=\"images/online.gif\" width=\"350\"/>\n",
+    "</div>\n",
+    "\n",
+    "One approach is to perform a test for drift every $W$ time-steps, using the $W$ samples that have arrived since the last test. In other words, that is to compare $\\{\\mathbf{X}_i\\}_{i=t}^{t+W}$ to $\\{\\mathbf{X}^{ref}_i\\}_{i=1}^{M}$. Such a strategy could be implemented using any of the offline detectors implemented in alibi-detect, but being both sensitive to slight drift and responsive to severe drift is difficult. If the window size $W$ is too large the delay between test-points hampers responsiveness to severe drift, but an overly small window is unreliable. This is demonstrated below, where the offline [MMD detector](methods/mmddrift.ipynb) is used to monitor drift in data sampled from a normal distribution $\\mathcal{N}\\left(\\mu,\\sigma^2 \\right)$, with the mean starting to drift from $\\mu=0$ to $\\mu=0.5$ at $t=40$.\n",
+    "\n",
+    "<div style=\"text-align: center\">\n",
+    "<img src=\"images/offline_drift_W2.gif\" width=\"700\"/>\n",
+    "<img src=\"images/offline_drift_W20.gif\" width=\"700\"/>\n",
+    "</div>\n",
+    "\n",
+    "\n",
+    "### Online drift detectors\n",
+    "\n",
+    "An alternative strategy is to perform a test each time data arrives. However the usual offline methods are not applicable because the process for computing p-values is too expensive, and it doesn’t account for correlated test outcomes when using overlapping windows of test data. Instead, [Alibi Detect](https://github.com/SeldonIO/alibi-detect) offers specialist online drift detectors:\n",
```

Comment:
Few things to mention here:
- "it doesn’t account for correlated test outcomes when using overlapping windows of test data" -> add something like "leading to miscalibrated detectors operating at an unknown False Positive Rate (FPR)" -> also make clear that well-calibrated FPR's are key to judge to severity of a drift detection and that in the absence of calibration, drift detection can be useless since there is no way of knowing what fraction are false positives
- Mention that the calibration procedure follows [Cobb et al.](https://arxiv.org/abs/2108.00883)